lib/sda/service.go

```diff
@@ -227,11 +231,9 @@ func (s *serviceStore) serviceByID(id ServiceID) (Service, bool) {
 // * Data: contains all the information of the request
 type ClientRequest struct {
 	// Name of the service to direct this request to
-	Service ServiceID `json:"service_id"`
-	// Type is the type of the underlying message
-	Type network.MessageTypeID `json:"type"`
```

Comment:
Taking away the wrapper, so we don't have the `ClientRequest` structure anymore. As it is a structure containing only `data[]`...